aurora/contracts/AuroraErc20FastBridge.sol

```diff
@@ -0,0 +1,391 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {IERC20 as IERC20_NEAR} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {AuroraSdk, NEAR, PromiseCreateArgs, PromiseResultStatus, PromiseWithCallback} from "@auroraisnear/aurora-sdk/aurora-sdk/AuroraSdk.sol";
+import "@auroraisnear/aurora-sdk/aurora-sdk/Borsh.sol";
+import "@auroraisnear/aurora-sdk/aurora-sdk/Utils.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/utils/Base64.sol";
+import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+import "./IEvmErc20.sol";
+
+contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlUpgradeable, PausableUpgradeable {
+    using AuroraSdk for NEAR;
+    using AuroraSdk for PromiseCreateArgs;
+    using AuroraSdk for PromiseWithCallback;
+    using Borsh for Borsh.Data;
+
+    bytes32 public constant PAUSABLE_ADMIN_ROLE = keccak256("PAUSABLE_ADMIN_ROLE");
+    bytes32 public constant UNPAUSABLE_ADMIN_ROLE = keccak256("UNPAUSABLE_ADMIN_ROLE");
+    bytes32 public constant CALLBACK_ROLE = keccak256("CALLBACK_ROLE");
+    bytes32 public constant WHITELIST_MANAGER = keccak256("WHITELIST_MANAGER");
+
+    uint64 constant BASE_NEAR_GAS = 10_000_000_000_000;
+    uint64 constant WITHDRAW_NEAR_GAS = 50_000_000_000_000;
+    uint64 constant INIT_TRANSFER_NEAR_GAS = 100_000_000_000_000;
+    uint64 constant UNLOCK_NEAR_GAS = 150_000_000_000_000;
+
+    uint128 constant NEAR_STORAGE_DEPOSIT = 12_500_000_000_000_000_000_000;
+
+    NEAR public near;
+    string public bridgeAddressOnNear;
+    string public auroraEngineAccountIdOnNear;
+    bool public isWhitelistModeEnabled;
+
+    //The Whitelisted Aurora users which allowed use fast bridge.
+    mapping(address => bool) whitelistedUsers;
+
+    //By the token address on near returns correspondent ERC20 Aurora token.
+    //[token_address_on_near] => aurora_erc20_token
+    mapping(string => IEvmErc20) registeredTokens;
+
+    //By the token account id on near and user address on aurora return the user balance of this token in this contract
+    //[token_address_on_near][user_address_on_aurora] => user_token_balance_in_aurora_fast_bridge
+    mapping(string => mapping(address => uint128)) balance;
```

Comment:
Is it possible to use the erc20 address instead of the nep141 to optimize read/write from/to the storage?